terraform/modules/cms/network.tf

```diff
@@ -264,4 +276,49 @@ resource "google_compute_backend_service" "cms_worker" {
   protocol         = "HTTPS"
   session_affinity = "NONE"
   timeout_sec      = "30"
-}
\ No newline at end of file
+}
+
+resource "google_compute_region_network_endpoint_group" "plateauview_geo" {
+  name                  = "${var.service_prefix}-plateauview-geo-neg"
+  network_endpoint_type = "SERVERLESS"
+  region                = "asia-northeast1"
+  cloud_run {
+    service = google_cloud_run_service.plateauview_geo.name
+  }
+}
+
+resource "google_compute_backend_service" "plateauview_geo" {
+  affinity_cookie_ttl_sec = "0"
+  enable_cdn              = true
+  cdn_policy {
+    signed_url_cache_max_age_sec = 7200
```

Comment:
Q.Is the CDN valid and correct?

Reply:
> I thought it will be necessary in the future, but it's not sure. So I'll remove it for now.
I'll remove it because of same reason.

Reply:
I fixed it:pray:
[Update cloudrun-geo.tf](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/5/commits/328df4c8e92cef0e31d0a09e383c3dfb930da324)